app/components/sortby-dropdown/component.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
You can set `tagName` (I think) here to `span` to avoid having one wrapping your template. It also avoids the `div` around everything, but you might want that?